llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.h

```diff
@@ -33,6 +37,7 @@ enum class DeviceLibExt : std::uint32_t {
 };
 
 using SYCLDeviceLibFuncMap = std::unordered_map<std::string, DeviceLibExt>;
```

Comment:
I think it could be moved to cpp file because it doesn't used anywhere else.

Reply:
I've moved the definition to the `.cpp` file and in the anonymous namespace.